### PR TITLE
[MM-13167] Fix for scroll lock when loading more posts

### PR DESCRIPTION
**Pull request by sudheerDev**
#### Summary
Removes state variable which causes a re-render trigger when loading posts

#### Ticket Link
[MM-13167](https://mattermost.atlassian.net/browse/MM-13167)

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [x] Ran `make test` to ensure unit and component tests passed

### CODE REVIEW

File: components/post_view/post_list.jsx [hunk 1]
```diff
@@ -269,7 +269,7 @@ export default class PostList extends React.PureComponent {
             return;
         }
 
-        if (this.state.isDoingInitialLoad || this.state.loadingPosts) {
+        if (this.state.isDoingInitialLoad || this.loadingPosts) {
```
Comment (saturninoabril) [inline components/post_view/post_list.jsx]: Don't you need to define and/or initialize `this.loadingPosts`, maybe at the constructor?

Comment (sudheerDev) [reply components/post_view/post_list.jsx]: Done. Added a default value

**Comment by enahum** [comment]
@sudheerDev will you be merging and cherry-picking after the PR gets tested, right?

**Comment by sudheerDev** [comment]
@enahum Yup thats the idea unless we plan to revert infinite scroll

**Comment by sudheerDev** [comment]
@enahum I will be merging this to master once build passes. I talked to carols and he will be trigger a build to community-daily after this is merged and then we can decide on cherrypick

**Pull request merged**